.github/workflows/lint.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint Code Base
```

Comment:
question: Will this block the PR or just show a warning when linter found some violation?

Reply:
No, it won't:
- for now we have only Travis CI check as required. Once This PR completed, I'll switch it to build GH action, but linting can be optional

![image](https://user-images.githubusercontent.com/827338/85325049-d116cc80-b47f-11ea-9fc7-057e5761802d.png)